lib/code_corps/policy/github_app_installation.ex

```diff
@@ -6,9 +6,7 @@ defmodule CodeCorps.Policy.GithubAppInstallation do
 
   alias CodeCorps.{GithubAppInstallation, User}
 
-  def create?(%User{} = user, %Ecto.Changeset{} = changeset),
-    do: changeset |> get_project |> owned_by?(user)
+  def create?(%User{} = user, params),
+    do: params |> get_project |> owned_by?(user)
 
```

Comment:
Minor, but would kill this line.

Reply:
removed newline and it's all one one line now, thx!